sdk/playwrighttesting/microsoft-playwright-testing/review/microsoft-playwright-testing.api.md

```diff
@@ -49,8 +49,8 @@ export type PlaywrightConfig = {
     use?: {
         connectOptions: BrowserConnectOptions;
     };
-    globalSetup?: string | string[];
-    globalTeardown?: string | string[];
+    globalSetup?: string | string[] | any;
```

Comment:
`string | string[] | any` is just `any` - are there any solutions that do not involve using `any`? 

I don't know your package but just a few ideas:
- Go back to only supporting `string` until your minimum playwright version is >= 1.49. This allows 
- Support `string | string[]` and detect the runtime version of playwright. If it's < 1.49 _and_ the input is an array, raise a useful error helping the user with guidance on how to resolve it


But I do worry about using `any` here as it does not allow us to ever narrow the types again